src/views/Pools/components/CakeVaultCard/FeeSummary.tsx

```diff
@@ -33,7 +34,7 @@ const FeeSummary: React.FC<FeeSummaryProps> = ({ stakingTokenSymbol, stakeAmount
   return (
     <>
       <Flex mt="24px" alignItems="center" justifyContent="space-between">
-        {tooltipVisible && tooltip}
+        {tooltipVisible && ReactDOM.createPortal(tooltip, document.getElementById('root'))}
```

Comment:
No need for Portal in this modal either.
Although second tooltip comes from UnstakingFeeCountdownRow and it needs Portal there to fix Gemran, so perhaps increasing z-index will work there so it displays properly both on card and on modal
![Screenshot 2021-05-21 at 14 05 08](https://user-images.githubusercontent.com/81824236/119128678-a0983380-ba3e-11eb-9cbc-fa2ef3c64505.png)

Reply:
I removed portal from these 2 and it works as expected. Can you verify?

Reply:
@memoyil removing portal from `unstaking fee until` breaks German again in the card, which was the whole point of this PR 😄 
Actually I think I found perfect solution for this problem that doesn't require on manually setting z-indices and adjusting each tooltip separately.... If you don't mind let's close this PR and I'll submit the generic fix for all tooltips. Don't spend all Friday fighting with tooltips 😅 
Appreciate your efforts and bringing it up 👍 

![Screenshot 2021-05-21 at 15 39 19](https://user-images.githubusercontent.com/81824236/119139799-1c4cad00-ba4c-11eb-99de-610394a36397.png)